packages/element/src/index-common.js

```diff
@@ -0,0 +1,20 @@
+import { createElement } from 'react';
```

Comment:
As a convention, we add dependencies blocks for import statements, in this case it will be:
```js
/**
 * External dependencies
 */
```